src/functions-and-arrays.js

```diff
@@ -1,73 +1,517 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 >= num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+function findLongestWord(arr) {
+  if (arr.length === 0) {
+    return null;
+  }
+  var longest = arr[0];
+  for (var i = 0; i < arr.length; i++) {
+    if (longest.length >= arr[i].length) {
```

Comment:
The conditional inside the for loop works good but you could refactor it a little bit better.

```suggestion
  if (arr[i].length > longest.length) {
      longest = arr[i];
    }
```

Removes the continue and else part to make it shorter and more concise.